src/Storage/Storage.Management/StorageAccount/SetAzureStorageLocalUser.cs

```diff
@@ -68,12 +68,12 @@ public class SetAzureStorageLocalUserCommand : StorageAccountBaseCmdlet
         public string HomeDirectory { get; set; }
 
         [Parameter(Mandatory = false,
-            HelpMessage = "The name of local user. The username must contain lowercase letters and numbers only. It must be unique only within the storage account.")]
+            HelpMessage = "Local user ssh authorized keys for SFTP.")]
         [ValidateNotNullOrEmpty]
         public PSSshPublicKey[] SshAuthorizedKey { get; set; }
 
         [Parameter(Mandatory = false,
-            HelpMessage = "The name of local user. The username must contain lowercase letters and numbers only. It must be unique only within the storage account.")]
+            HelpMessage = "The permission scopes of the local user.")]
```

Comment:
I would suggest also mention how to create the parameter. 
refer the sample: https://learn.microsoft.com/en-us/powershell/module/az.storage/set-azstorageblobinventorypolicy?view=azps-9.4.0#-rule

It should be created with cmdlet : New-AzStorageLocalUserPermissionScope

Reply:
Done